index.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (base, country, language, callback) {
         var out = {};
         async.eachSeries(files, function (ent, next) {
             findAllBundlesFromRoot(ent, function (err, o) {
-                out[path.relative(base, ent).replace(/(.*)\/(.*)/, "$2-$1")] = o;
+                out[path.relative(base, ent).replace(new RegExp('(.*)\\' + path.sep + '(.*)'), "$2-$1")] = o;
```

Comment:
user-supplied? I don't see the use case -- configurability is a footgun.

But I think it can be made universal so things always end up in unix form. I'd prefer to keep the internals unix-form as much as possible, so push the win32 corrections out to the edge as far as possible.